mesonbuild/wrap/wrap.py

```diff
@@ -226,16 +226,21 @@ def get_file(self):
             self.apply_patch()
 
     def get_git(self):
+        # git doesn't support directly cloning shallowly for commits,
+        # so we follow https://stackoverflow.com/a/43136160
+        subprocess.check_call(['git', 'init', self.directory], cwd=self.subdir_root)
+        subprocess.check_call(['git', 'remote', 'add', 'origin', self.wrap.get('url')],
+                               cwd=self.dirname)
```

Comment:
__[Flake8]__ [E127] continuation line over-indented for visual indent ([link](https://sider.review/gh/repos/19784232/pulls/5932#issue-4580790))

<sub>posted by Sider</sub>

Reply:
This should be on the same column as the `[`; it is an argument to the function and thus related to its opening parenthesis.

Reply:
Thank you.  I fixed the indentation and `fixup`ed the last commit.